popup.js

```diff
@@ -0,0 +1,183 @@
+const projects = [
+  {
+    title: 'TONIC 1',
+    prof: ['Canopy', 'back end Dev', '2015'],
+    src: 'images/SnapshootPortfolio.png',
+    description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
+    technologies: ['html', 'css', 'javascript', 'Github', 'Ruby', 'Bootstrap'],
+    seeLiveLink: '#',
+    seeSourceLink: '#',
+  },
+  {
```

Comment:
- [ ] In the previous projects, we have been adding the projects manually using HTML. However, as the project requirements suggest, you should put the following into consideration ☺️ .
      - You need to use a JavaScript array to store all of the information for all projects.
      - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
      - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
 - This means that even the project cards should be added dynamically. Then after all that you can implement the modal popup.